src/main/java/net/engio/mbassy/listener/RepeatedFilters.java

```diff
@@ -0,0 +1,23 @@
+package net.engio.mbassy.listener;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+/**
+ * The repeated filters annotation is used to add multiple {@link Filter}s to
+ * a single {@link ElementType#ANNOTATION_TYPE annotation type} when using
+ * "inherited filters".
+ *
+ * @see Filter
+ */
+@Retention(RetentionPolicy.RUNTIME)
+@Target(ElementType.ANNOTATION_TYPE)
+public @interface RepeatedFilters {
```

Comment:
This is required due to compiling against Java 6, and can be removed in favour of `@Repeatable` if mbassador is updated to Java 8.